backend/app/views/spree/admin/shared/preference_fields/_boolean.html.erb

```diff
@@ -0,0 +1,13 @@
+<% label = local_assigns[:label].presence %>
+<% html_options = local_assigns[:html_options] || {} %>
+
+<div class="field">
+  <% if local_assigns[:form] %>
+    <%= form.check_box attribute, html_options %>
+    <%= form.label attribute, label %>
+  <% else %>
+    <%= hidden_field_tag name, 0 %>
```

Comment:
Turns out this would be tricky, because we would need to rewrite all forms rendering these fields. In terms of the calculator fields this means to rewrite the calculator forms. I think we should tackle this in a separate PR. WDYT?